### ISSUE TITLE
[flang][OpenMP] need an option to suppress 'warning: OpenMP support for version 50 in flang is still incomplete'

### ISSUE THREAD

**Issue by ye-luo** [llvm/llvm-project#195921]
Without high enough version setting. I got
```
/home/yeluo/opt/q-e/q-e-repo/UtilXlib/rocblas.f90:400:31: error: USE_DEVICE_ADDR clause is not allowed on directive TARGET DATA in OpenMP v3.1, try -fopenmp-version=50
```
After adding `-fopenmp-version=50`, I got massive warning
```
warning: OpenMP support for version 50 in flang is still incomplete
```
Need a way to suppress this warning.
